AudioDeviceExample/ViewController.swift

```diff
@@ -256,8 +258,6 @@ extension ViewController: CXProviderDelegate {
 
     func provider(_ provider: CXProvider, perform action: CXStartCallAction) {
         NSLog("provider:performStartCallAction:")
-
-        audioDevice.isEnabled = false
```

Comment:
Is this change needed?

Reply:
I was following the same updates and audio device pattern from the test app

Reply:
AudioDevice should be disabled by default. should we add an assert instead?

Reply:
I would prefer showing the SDK users that they should now only use the CallKit provider callback to enable/disable audio device. We do have test for the default value in the source.

Reply:
I didn't realize this is audio device example sample app.